docs/develop/which-tools.mdx

```diff
@@ -2,15 +2,15 @@ import ThemedImage from '@theme/ThemedImage';
 
 # Which tools to use
 
-There are a variety of tools in the Terra Development Suite. Use this guide to find the right tool for your project.
+There are a variety of tools in the Terra Development Suite. This guide will help you to find the right tool to use for your project.
```

Comment:
"to find" and "to use" are unnecessary infinitives here. 

Might I suggest the following:
"...will help you find the right tools for your project."